### PR TITLE
Client fixes for empty amount bug

### PR DESCRIPTION
**Pull request by patrickxb**

### CODE REVIEW

**Comment by patrickxb** [comment]
@mlsteele you might not be happy with this, but these empty amount issues keep coming in despite all server fixes.

it's something when the user is accepting the disclaimer and obviously a crappy experience to get an error right after that, so i'd really like to get this in just to get rid of those errors.  we can investigate the cause later.

File: go/stellar/stellar.go [hunk 1]
```diff
@@ -1606,7 +1606,9 @@ func FormatAmountWithSuffix(amount string, precisionTwo bool, simplify bool, suf
 
 func FormatAmount(amount string, precisionTwo bool, rounding FmtRounding) (string, error) {
 	if amount == "" {
-		return "", errors.New("empty amount")
+		// purists might complain here, but I really just want
+		// to get past this bug
+		amount = "0"
```
Comment (mlsteele) [inline go/stellar/stellar.go]: as long as this isn't used for anything to do with money it seems fine

**Comment by mlsteele** [comment]
These keep coming in meaning we get logs from users? Let's add stack traces to the logs. Do we get any from admins?

**Comment by patrickxb** [comment]
From users only.

**Pull request merged**